GameServerLib/GameObjects/AttackableUnits/AttackableUnit.cs

```diff
@@ -30,15 +31,15 @@ public class AttackableUnit : GameObject, IAttackableUnit
         public IReplication Replication { get; protected set; }
 
         public AttackableUnit(
-            Game game,
+            IGame game,
```

Comment:
Why not just keep the implementation? I don't understand your change, and it adds an unnecessary cast.
```suggestion
            Game game,
```

Reply:
Used by scripts that spawn minions. Spawning requires game instance.

Reply:
Where scripts use AttackableUnit? they should use only IAttackableUnit or similar things.

Reply:
It uses IMinion: `IMinion Jack = new Minion(owner.RefGame, owner, spell.X, spell.Y, "ShacoBox", "ShacoBox", sightrange, 0);
                owner.RefGame.ObjectManager.AddObject(Jack);
                Jack.SetVisibleByTeam(owner.Team, true);`

Reply:
I'm not sure this is in the cur build, that shouldn't be the case, the API should wrap this.

Reply:
In my PR, it uses IMinion.

Reply:
ok, so we should open issue about API wrapping. There should be API call like `AddMinion(...)` etc.

Reply:
I agree, but would it come before or after my PR?

Reply:
After, but I don't see the minion creation code here: https://github.com/LeagueSandbox/LeagueSandbox-Default/pulls, so maybe you didn't PRed the submodule.

Reply:
Fix here as suggested, It should be possible after introducing the API `AddMinion` function.